plugin/pkg/admission/broker/authsarcheck/admission_test.go

```diff
@@ -240,7 +240,7 @@ func TestAdmissionBroker(t *testing.T) {
 		}
 		kubeInformerFactory.Start(wait.NeverStop)
 
-		err = handler.(admission.MutationInterface).Admit(admission.NewAttributesRecord(tc.broker, nil, servicecatalog.Kind("ClusterServiceBroker").WithVersion("version"), tc.broker.Namespace, tc.broker.Name, servicecatalog.Resource("clusterservicebrokers").WithVersion("version"), "", admission.Create, false, tc.userInfo))
+		err = handler.(admission.MutationInterface).Admit(admission.NewAttributesRecord(tc.broker, nil, servicecatalog.Kind("ClusterServiceBroker").WithVersion("version"), tc.broker.Namespace, tc.broker.Name, servicecatalog.Resource("clusterservicebrokers").WithVersion("version"), "", admission.Create, false, tc.userInfo), nil)
```

Comment:
left it as nil unless and until we figure out what to do with it

Reply:
should be fine.